packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -71,4 +85,8 @@ export default class QuanticResultLink extends LightningElement {
       this.headless.buildInteractiveResult
     );
   };
+
+  get anchorHTMLElement() {
+    return this.template.querySelector('a');
+  }
```

Comment:
Just like the facet values, you find it cleaner to have a separate method to do this instead of just having this inline of the setFocus method?
I'm curious to hear your thoughts on this.

Reply:
I feel, both way could convey the information to the reader. In this case, it is easier to read when all the related action are put in one place. And since we are using other format for `setFocus` in other components, changing this one to other format.